startupscript/butane/aws/metadata-utils.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# metadata-utils.sh defines helper functions for aws ec2 tags. This script is intended to be sourced from other scripts
+# to retrieve or modify instance tags. It is run on the VM host. AWS CLI is not installed on Flatcar VM by default so
+# we are running the AWS CLI in a container.
+
+# Defines a function to retrieve an instance tag set on the VM.
+function get_metadata_value() {
+  if [[ -z "$1" ]]; then
+    echo "usage: get_metadata_value <tag>"
+    exit 1
+  fi
+  local tag_key=vwbapp:"$1"
+
+  INSTANCE_ID="$(wget -q -O - http://169.254.169.254/latest/meta-data/instance-id)"
```

Comment:
Since this line is running directly on the VM host, it can already start using IMDSv2. See
https://github.com/nfoen/workbench-app-devcontainers/blob/master/startupscript/aws/vm-metadata.sh

I'm about to make a PR in this repo anyways, so I can also make the update in this script as well.

Reply:
done